tests/local_vespa/test_app/schemas/family_document.sd

```diff
@@ -11,12 +11,12 @@ schema family_document {
     }
 
     field family_name_bolding type string {
-        indexing: input family_name_index | index
+        indexing: input family_name_index | summary | index
         bolding: true
     }
 
     field family_description_bolding type string {
-        indexing: input family_description_index | index
+        indexing: input family_description_index | summary | index
```

Comment:
Any reason to hang onto the index attribute for these when they come from an index field?

Reply:
i don't know, but figured that hanging onto it is probably the safest thing to do as I imagine the resource usage implications will be minor

Reply:
That makes sense 👍